### DISCUSSION TITLE
下载源可否多个github下载源呢？

### DISCUSSION THREAD

**Discussion by robinsletter** [Meloong-Git/PCL#3625] [提问] [answered]
像有些mod只有github上才能下载

**Comment by hejiehao** [comment]
不能

**Comment by allMagicNB** [comment]
你说，谁会来整理 GitHub 上的 Mod 呢？

**Comment by burningtnt** [comment]
不能

**Comment by Deep-Dark-Forest** [comment]
你怎么知道哪个 github repo 里面装的是模组？

**Comment by zkitefly** [comment, marked as answer]
https://github.com/Hex-Dragon/PCL2/issues/2288

> GitHub 没有固定 API，且国内访问情况极差，不会考虑制作此功能，抱歉

**Comment by wuliaodexiaoluo** [comment]
> 像有些mod只有github上才能下载

因为 GitHub API（api.github.com） 返回的信息没有统一标准和规范，通过搜索返回的信息又相当杂，无法制作
